packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -408,6 +411,25 @@ class FlutterDriverExtension {
     return GetOffsetResult(dx: globalPoint.dx, dy: globalPoint.dy);
   }
 
+  Future<DiagnosticsTreeResult> _getDiagnosticsTree(Command command) async {
+    final GetDiagnosticsTree diagnosticsCommand = command;
+    final Finder finder = await _waitForElement(_createFinder(diagnosticsCommand.finder));
+    final Element element = finder.evaluate().single;
+    DiagnosticsNode diagnosticsNode;
+    switch (diagnosticsCommand.diagnosticsType) {
+      case DiagnosticsType.renderObject:
+        diagnosticsNode = element.renderObject.toDiagnosticsNode();
+        break;
+      case DiagnosticsType.widget:
+        diagnosticsNode = element.toDiagnosticsNode();
```

Comment:
Should this be `DiagnosticsType.element` then?  And the associated method called `getElementDiagnostics`?

Reply:
This is consistent with the terminology we use in the Widget Inspector for these two trees.